lib/core/test/integration/Test/Integration/Framework/TestData.hs

```diff
@@ -241,9 +243,14 @@ errMsg403InputsDepleted = "I had to select inputs to construct the requested\
     \ transaction. Unfortunately, one output of the transaction depleted all\
     \ available inputs. Try sending a smaller amount."
 
-errMsg403InvalidTransaction :: String
-errMsg403InvalidTransaction = "I can't process this payment because it contains at least\
-    \ one payment output of value 0. This isn't supported by the current core nodes."
+errMsg403InvalidTransactionZeroAmtOutput :: String
+errMsg403InvalidTransactionZeroAmtOutput = "I can't validate coin selection because transaction contains\
+    \ at least one payment output of value 0."
+
+_errMsg403InvalidTransactionWrongInpsOuts :: MaxInpsOrOuts -> String
+_errMsg403InvalidTransactionWrongInpsOuts (MaxInpsOrOuts maxNumInps maxNumOuts) =
+    "I can't validate coin selection because transaction has either\
```

Comment:
Change it in such a way that transaction is not mentioned. This is because we use the same error both when constructing transaction and when estimating fee